ts/parts/Pointer.ts

```diff
@@ -1283,15 +1287,21 @@ class Pointer {
         // Normalize before the 'if' for the legacy IE (#7850)
         e = this.normalize(e);
 
-        if ((e as any).button !== 2) {
+        this.onContainerMouseMove(e);
```

Comment:
It seems a little strange to call `onContainerMouseMove()` from the `onContainerMouseDown` function... Why are we doing that? Do we know the side effects? Are there parts of the `onContainerMouseMove` function that we can call directly instead?

At the least, we should have a comment explaining why we do this strange call.

Reply:
The event order between Firefox and Chrome after click scrolling is different. Firefox does not fire reliable a move event after releasing the click wheel. Chrome fires always down and click.